docs/miscellaneous.rst

```diff
@@ -56,6 +56,22 @@ So for the following contract snippet::
 
 The position of ``data[4][9].b`` is at ``sha3(uint256(9) . sha3(uint256(4) . uint256(1))) + 1``.
 
+********************
+When Solidity Throws
+********************
+
+Solidity contract throws an exception for unhealthy operations such as
+
+- division by zero
+- modulo by zero
+- out-of-bounds access on a fixed bytes value
+- out-of-bounds access on an array
+- a positive value seen by a function without ``payable`` mofifier except when the function is a library function
```

Comment:
I think "positive value" should be more descriptive, maybe: "Ether transfer seen"